canvas_modules/common-canvas/src/common-properties/components/title-editor/title-editor.jsx

```diff
@@ -92,6 +94,24 @@ class TitleEditor extends Component {
 		this.props.setTitle(newTitle);
 	}
 
+	createTitleActions() {
+		if (this.props.titleInfo) {
+			this.actionFactory = new ActionFactory(this.props.controller);
+			const actions = [];
+			this.props.titleInfo.Title.actionIds().action_refs.forEach((actionRef, idx) => {
```

Comment:
Will actionIds() always return a list of actions or an empty list?

Reply:
Yes it will always return a list. I just updated the code to fix TitleInfo to return the actions instead of the action_refs and to not include a padding if an empty list is defined